.eslintrc

```diff
@@ -70,6 +70,7 @@
     "unicorn/prefer-date-now": "error",
     "unicorn/prefer-dom-node-append": "error",
     "unicorn/prefer-dom-node-remove": "error",
+    "unicorn/prefer-includes": "error",
```

Comment:
Does this mean that we could/should remove the following existing rule, since they seem to be similar?
https://github.com/mozilla/pdf.js/blob/341a0b6d477d2909fcb14bcbfdf0d2fd37406cb0/.eslintrc#L48

Based on the documentation, see https://firefox-source-docs.mozilla.org/code-quality/lint/linters/eslint-plugin-mozilla/rules/use-includes-instead-of-indexOf.html, it does seem that the `mozilla` rule may be less "complete" anyway.

Reply:
Nice find; I hadn't noticed that one yet. You're right, the new `unicorn` rule should do the same, but in addition to the Mozilla rule also catch e.g. `lastIndexOf()` usage.